jerry-core/parser/js/js-parser.c

```diff
@@ -2291,19 +2291,20 @@ parser_parse_source (const uint8_t *arg_list_p, /**< function argument list */
                      const uint8_t *source_p, /**< valid UTF-8 source code */
                      size_t source_size, /**< size of the source code */
                      uint32_t parse_opts, /**< ecma_parse_opts_t option bits */
-                     parser_error_location_t *error_location_p) /**< error location */
+                     parser_error_location_t *error_location_p /**< error location */
+#ifndef CONFIG_DISABLE_ES2015_MODULE_SYSTEM
+                     ,
+                     parser_module_node_t *module_node_p, /**< module node that contains imports */
+                     parser_module_node_t *export_node_p  /**< module node that contains exports */
```

Comment:
Uhh, this is a bit ugly! Cannot these values can be part of JERRY_CONTEXT, which could contain the current {import, export}_module_t? See how the JERRY_CONTEXT (vm_top_context) works.